tests/Start-DbaXESession.Tests.ps1

```diff
@@ -90,22 +91,34 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
         It "works when -StopAt is passed" {
             $StopAt = (Get-Date).AddSeconds(10)
             Start-DbaXESession $server -Session $dbatoolsciValid.Name -StopAt $StopAt -WarningAction SilentlyContinue
+            $dbatoolsciValid.Refresh()
             $dbatoolsciValid.IsRunning | Should Be $true
             (Get-DbaAgentJob -SqlInstance $server -Job "XE Session STOP - dbatoolsci_session_valid").Count | Should -Be 1
             $stopSchedule = Get-DbaAgentSchedule -SqlInstance $server -Schedule "XE Session STOP - dbatoolsci_session_valid"
             $stopSchedule.ActiveStartTimeOfDay.ToString('hhmmss') | Should -Be $StopAt.TimeOfDay.ToString('hhmmss')
             $stopSchedule.ActiveStartDate | Should -Be $StopAt.Date
+            Start-Sleep -Seconds 11
```

Comment:
is the 11 calculated in some way or is it just "let's wait for 11 seconds, to make sure?"